samples/xmlserializergenerator-instructions.md

```diff
@@ -0,0 +1,46 @@
+# Using Microsoft XmlSerializer Generator on .Net Core
+
+An XmlSerializer generates serialization code and a serialization assembly for each type every time an application run during the Xml Serialization. To improve the performance, we can use Microsoft XmlSerializer Generator to generate Xml Serialization code for types in specified project and compile it into an assembly in advance. These assemblies can then run with the application.
+
+## Prerequisition
+
+You need install the followings on your machine:
+* [.NET Core SDK daily builds](https://github.com/dotnet/cli#installers-and-binaries) 
+* [.NET Core runtime daily builds](https://github.com/dotnet/core-setup#daily-builds). 
+
+You can validate your .NET Core SDK version by typing `dotnet --info`.
+
+## Instructions
+
+Here is the step by step how to create a dotnet project and install the XmlSerializer Generator on the project.
+
```

Comment:
Here are the step by step instructions how to create a dotnet project and install the XmlSerializer Generator for that project.